packages/engine/src/simulation/package/state/packages/behavior_execution/mod.rs

```diff
@@ -34,7 +34,7 @@ pub type BehaviorIdInnerDataType = u16;
 pub type BehaviorIndexInnerDataType = f64;
 
 pub struct Creator {
-    experiment_config: Option<Arc<ExperimentConfig>>,
+    _experiment_config: Option<Arc<ExperimentConfig>>,
```

Comment:
This one actually feels weird to me, I think we can just delete these. Packages can decide what they want to keep in the Creator so no need to keep it for consistency